tests/testthat/test_Study_Report.R

```diff
@@ -0,0 +1,39 @@
+# Install and load the `testthat` package
+source(testthat::test_path("testdata/data.R"))
+
+# Write tests for Study_Report function
+test_that("Study_Report function works as expected", {
+  snapshot <- Make_Snapshot(lData = lData)
+
+  # Create temporary output file path
+  # tmp_outpath <- tempfile(fileext = ".html")
+
+  # Test the function with different report types
+  expect_error(
+    Study_Report(snapshot, strOutpath = tmp_outpath, strReportType = "person")
+  )
+
+  expect_error(
+    Study_Report(snapshot, strOutpath = tmp_outpath, strReportType = c("site", "QTL"))
+  )
+
+
+  # Study_Report(snapshot, strOutpath = tmp_outpath, strReportType = "site")
+  # expect_true(
```

Comment:
Don't worry about testing that the report renders or anything like that - I'd focus more on the parameters/checks in `Study_Report()`.